roles/confluent.test/molecule/rbac-scram-custom-rhel/certificate-hosts

```diff
@@ -1,3 +1,4 @@
+zookeeper:zookeeper1
```

Comment:
shouldnt need this since zookeeper_ssl_enabled will default to false
same goes for other scenarios